File: donghwan/week1/12930/12930.py
def solution(s):
    index = 0
    answer = ""
    for i in range(len(list(s))):
        if s[i] == " ":
            index = 0
            answer += " "
        elif index % 2 == 0:
            index += 1
            answer += s[i].upper()
        else:
            index += 1
            answer += s[i].lower()
            
    return answer

File: donghwan/week1/12930/12930.java
import java.util.*;

class Solution {
    public String solution(String s) {
        StringBuilder builder = new StringBuilder();
    
        int index = 0;
        for(String str : s.split("")) {
            if(str.equals(" ")) {
                index = 0;
                builder.append(" ");
                continue;
        }
            if(index % 2 == 0) {
                builder.append(str.toUpperCase());
            } else {
                builder.append(str.toLowerCase());
            }
            index++;
        }
    return builder.toString();
    }
}
